api/v1/swagger.yaml

```diff
@@ -1179,11 +1179,11 @@ paths:
       security:
         - jwt: []
       tags: [ collectors ]
-      summary: Create a new collector
+      summary: Access restricted to Refocus Collector only.
       description: >-
-        Create a new collector. This request may only be issued by the refocus-collector register command.
-        The new record is automatically write-protected for just the user who created it. The default status: STOPPED.
-        isRegistered is true. System fills in the timeStamp fields.
+        Access restricted to Refocus Collector only. This request is issued by the refocus-collector REGISTER command.
```

Comment:
The description for this one should ONLY be "Access restricted to Refocus Collector only."